{"error":{"root_cause":[{"type":"parsing_exception","reason":"Unknown key for a VALUE_STRING in [q].","line":1,"col":7}],"type":"parsing_exception","reason":"Unknown key for a VALUE_STRING in [q].","line":1,"col":7},"status":400}